src/msw/display.cpp

```diff
@@ -56,7 +56,7 @@ class wxDisplayImplSingleMSW : public wxDisplayImplSingle
     virtual wxRect GetClientArea() const wxOVERRIDE
     {
         RECT rc;
-        SystemParametersInfo(SPI_GETWORKAREA, 0, &rc, 0);
+        wxSystemParametersInfo(SPI_GETWORKAREA, 0, &rc, 0);
```

Comment:
As with `wxScreenDC` above, I think we should avoid changing this code and let it continue use the primary monitor.